modules/customization/proc-customize-rhdh-sidebar-menuitems.adoc

```diff
@@ -102,13 +102,14 @@ dynamicPlugins:
 dynamicPlugins:
   frontend:
     default.main-menu-items: # key for configuring static main menu items
-      default.<menu_item_name>: # key of the menu item configuration. `default.` prefix is required for a main menu item key <1>
-        parent: my_menu_group # optional, specifies the parent menu item for this item
-        priority: 10 # optional, specifies the order of this menu item within its menu level
-      default.<menu_group_parent_item_name>: # must be configured if it is specified as the parent of any menu items. `default.` prefix is required for a main menu item key
-        icon: my_menu_group_icon # required for parent menu items, defines the icon for the menu group
-        title: my_menu_group_title # required for parent menu items, defines the icon for the menu group
-        priority: 100 # optional, specifies the order of the parent menu item in the sidebar
+      menuItems:
+        default.<menu_item_name>: # key of the menu item configuration. `default.` prefix is required for a main menu item key <1>
+          parent: my_menu_group # optional, specifies the parent menu item for this item
+          priority: 10 # optional, specifies the order of this menu item within its menu level
+        default.<menu_group_parent_item_name>: # must be configured if it is specified as the parent of any menu items. `default.` prefix is required for a main menu item key
+          icon: my_menu_group_icon # required for parent menu items, defines the icon for the menu group
+          title: my_menu_group_title # required for parent menu items, defines the icon for the menu group
+          priority: 100 # optional, specifies the order of the parent menu item in the sidebar
```

Comment:
I believe using this notation would bring more clarity and readability. Additionally you would rather explain to add the menu items in the `default` section rather than using a `default.` prefix. 

```suggestion
      menuItems:
        default:
          <menu_item_name>: # key of the menu item configuration. `default.` prefix is required for a main menu item key <1>
            parent: my_menu_group # optional, specifies the parent menu item for this item
            priority: 10 # optional, specifies the order of this menu item within its menu level
          <menu_group_parent_item_name>: # must be configured if it is specified as the parent of any menu items. `default.` prefix is required for a main menu item key
            icon: my_menu_group_icon # required for parent menu items, defines the icon for the menu group
            title: my_menu_group_title # required for parent menu items, defines the icon for the menu group
            priority: 100 # optional, specifies the order of the parent menu item in the sidebar
```